spec/presenters/show_presenter_spec.rb

```diff
@@ -283,5 +283,31 @@ def mock_document_app_helper_url *args
       end
     end
   end
+
+  describe 'deprecated methods' do
+    before do
+      allow(Deprecation).to receive(:warn)
```

Comment:
If their being deprecated is part of the point, shouldn't this be a `before :each { expect ... }`?